praw/config.py

```diff
@@ -33,7 +33,7 @@ class Config:
     def _config_boolean(item):
         if isinstance(item, bool):
             return item
-        return item.lower() in {"1", "yes", "true", "on"}
+        return str(item).lower() in {"1", "yes", "true", "on"}
```

Comment:
Why is `str` added?

Reply:
I thought that this could throw an error if a non-str object was used, such as an int.

Reply:
I don't think it's possible for a non-string value to be passed in a config file.